x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -298,15 +348,70 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
           dataViewId === null
             ? []
             : getFiltersForDSLQuery(job.datafeed_config.query, dataViewId, job.job_id),
-      });
+        ...(withWindowParameters
+          ? {
+              wp: { bMin, bMax, dMin, dMax },
+            }
+          : {}),
+      };
+    };
+
+    const generateDiscoverUrl = async () => {
+      const pageState = await generateRedirectUrlPageState();
+      const url = await discoverLocator.getRedirectUrl(pageState);
 
       if (!unmounted) {
         setOpenInDiscoverUrl(url);
       }
     };
 
+    const generateLogRateAnalysisUrl = async () => {
+      if (props.anomaly.source.function_description !== 'count') {
```

Comment:
Take a look at the `visitor_rate_ecs` job in our Apache access module - the detector is `non_zero_count` but on a `summary_count_field_name` that is generated via an agg which is doing distinct count. This can lead to some results which can appear puzzling - for example opening log rate analysis from the anomaly here:

<img width="716" alt="image" src="https://github.com/elastic/kibana/assets/7405507/1504006b-7575-4fab-82fa-2f8092480b2a">

displays this:

<img width="850" alt="image" src="https://github.com/elastic/kibana/assets/7405507/b64112db-31c3-4330-9bc5-27110e6d60bd">

On first glance you wonder if the correct bucket is selected - it is, but it can be misleading as the anomaly is really looking at `distinct_count` rather than event rate.

It might be worth hiding the action if the datafeed is using an agg? I think we do some similar checks in the Single Metric Viewer to ensure the correct series is plotted.